test/unit/CMakeLists.txt

```diff
@@ -85,13 +85,12 @@ add_dependencies(unit.exe unit.combinatorial.exe )
 glob_unit("unit" ${unit_root} "module/combinatorial/*.cpp" )
 
 if(EVE_USE_BOOST)
+
+if(NOT DEFINED EVE_TEST_NO_BESSEL)
 add_custom_target(unit.bessel.exe         )
 add_dependencies(unit.exe unit.bessel.exe )
 glob_unit("unit" ${unit_root} "module/bessel/*.cpp" )
-
-# add_custom_target(unit.complex.exe         )
-# add_dependencies(unit.exe unit.complex.exe )
-# glob_unit("unit" ${unit_root} "module/complex/*.cpp" )
+endif()
```

Comment:
If the test doesn't work, let's disable it in code by looking at spy? that macro in targets is not visible most of the time and is annoying to build

Reply:
No. Complex and quaternion has been removed ages ago but didn't cleaned up the cmake